.obsidian/core-plugins-migration.json

```diff
@@ -25,5 +25,6 @@
   "workspaces": false,
   "file-recovery": true,
   "publish": false,
-  "sync": false
+  "sync": false,
+  "bookmarks": true
```

Comment:
@Lawrencium77 any idea what's going on here?

Reply:
No idea tbh, don't think it's a big deal